### ISSUE TITLE
go wrong for quick usage with  prompt massege "got response code 401 from Unsplash - Unauthorized"

### ISSUE THREAD

**Issue by orin189** [MehediH/Bulksplash#52]
I try to use Bulksplash by the quick usage, but went wrong. and i use the Unsplash API Key go well through the prompt interface.

==quick usage===
F:\TEMP\tmp1>bulksplash --a=50 --d=basket --c=https://unsplash.com/collections/9454911 --f --j --k=0z_vd1QxOCCqOWJHA25NXHwETXIJ%
🤖 Welcome to Bulksplash! (Powered by Unsplash.com)
🔰 Downloading 20 images from:
🚨 Something went wrong, got response code 401 from Unsplash - Unauthorized
========

system: windows 10 professional 64bit
![图片](https://user-images.githubusercontent.com/5311125/231337505-87edb282-8d1f-4e1f-9884-ff9bed03a9f4.png)

**Comment by thanasistrisp** [comment]
It is fixed with #51. Try to build it yourself by cloning the repository. To work from there (https://www.npmjs.com/package/bulksplash) @MehediH should push an update.

**Comment by bytesweep** [comment]
I am also encountering the same error I am wondering if there are any adjustments I need to make? This is from a download/install from today. Windows 11

Worked when I cloned the repo and it was successful running off of that as reported by @thanasistrisp

**Comment by MichaelAlanBaker** [comment]
I am also having the same problem... and I'm a little confused about how to correctly use the API my access key and secret key... Do I just provide the access key and it should work without authentication or to I need to concatenate them both some how?

**Comment by thanasistrisp** [comment]
> I am also having the same problem... and I'm a little confused about how to correctly use the API my access key and secret key... Do I just provide the access key and it should work without authentication or to I need to concatenate them both some how?

For now, just clone repository and run:
```
node cli.js --a=50 --d=basket -c=https://unsplash.com/collections/9454911 --f --j --k=API_KEY
```
Key should be passed each time.

**Comment by bytesweep** [comment]
> > I am also having the same problem... and I'm a little confused about how to correctly use the API my access key and secret key... Do I just provide the access key and it should work without authentication or to I need to concatenate them both some how?
> 
> 
> 
> For now, just clone repository and run:
> 
> ```
> 
> node cli.js --a=50 --d=basket -c=https://unsplash.com/collections/9454911 --f --j --k=API_KEY
> 
> ```
> 
> Key should be passed each time.

This is the method that worked for me. I kept it easy. Cloned it, the directory I just called images and left it in the base of the clone and it worked just fine.

**Comment by orin189** [comment]
thanks so much @thanasistrisp and @MehediH ，now quick usage can work well. the issue can be closed. 

> It is fixed with #51. Try to build it yourself by cloning the repository. To work from there (https://www.npmjs.com/package/bulksplash) @MehediH should push an update.

**Comment by kloodhu** [comment]
Could someone stepbystep, what should we do for using the new version? sorry, aim very noob for this! thank you!
